ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py

```diff
@@ -3659,6 +3659,40 @@ def test_paddle_tensor_tanh(
     )
 
 
+# tanh_
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="tanh_",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+    ),
+)
+def test_paddle_tensor_tanh_(
+    dtype_and_x,
+    frontend_method_data,
+    init_flags,
+    method_flags,
+    frontend,
+    on_device,
+    backend_fw,
+):
+  input_dtype, x = dtype_and_x
```

Comment:
There's a linting issue here. Can you please fix this?

Reply:
I also need to see the latest test results. Can you merge main branch in your branch and run the tests again?

Reply:
solved the lint error and tested with latest main branch pull. it is passing all the tests 😄 

![image](https://github.com/unifyai/ivy/assets/83540902/3c09e45d-8060-4346-9444-2b04b12638ff)

mergeeee.. 😄 🚀

Reply:
@vedpatwardhan this is also one of those PRs where CI fails but tests pass locally 🤔 . What could be the issue?